packages/web/src/pages/upload-page/fields/AccessAndSaleField.tsx

```diff
@@ -477,7 +526,15 @@ export const AccessAndSaleField = (props: AccessAndSaleFieldProps) => {
       )
     }
 
-    let selectedValues = []
+    let selectedValues: (
+      | {
+          label: string
+          icon: IconComponent
+          ['data-testid']?: string
+          key?: string
```

Comment:
is there not a type for this somewhere in/around `ContextualMenu` or `SelectedValue`?

Reply:
Good call. Updated to `SelectedValue`